packages/nc-gui-v2/components/dlg/TableCreate.vue

```diff
@@ -38,17 +31,17 @@ const { table, createTable, generateUniqueTitle, tables, project } = useTable(as
   dialogShow.value = false
 })
 
-const prefix = computed(() => project?.value?.prefix || '')
+// const prefix = computed(() => project?.value?.prefix || '')
 
 const validateDuplicateAlias = (v: string) => {
   return (tables?.value || []).every((t) => t.title !== (v || '')) || 'Duplicate table alias'
 }
-const validateLeadingOrTrailingWhiteSpace = (v: string) => {
-  return !/^\s+|\s+$/.test(v) || 'Leading or trailing whitespace not allowed in table name'
-}
-const validateDuplicate = (v: string) => {
-  return (tables?.value || []).every((t) => t.table_name.toLowerCase() !== (v || '').toLowerCase()) || 'Duplicate table name'
-}
+// const validateLeadingOrTrailingWhiteSpace = (v: string) => {
+//   return !/^\s+|\s+$/.test(v) || 'Leading or trailing whitespace not allowed in table name'
+// }
+// const validateDuplicate = (v: string) => {
+//   return (tables?.value || []).every((t) => t.table_name.toLowerCase() !== (v || '').toLowerCase()) || 'Duplicate table name'
+// }
```

Comment:
Can't this just be removed instead of commenting it out?

Reply:
not sure if it is required or not , so I just commented for the time being... will confirm and remove it